package.json

```diff
@@ -23,6 +23,7 @@
     "svg": "gulp --gulpfile build/gulpfile.js svg-sprite",
     "test": "npm run lint",
     "watch": "watch 'npm run build' ./src -d",
+    "watch-library": "watch 'npm run build-library-assets' ./src -d",
```

Comment:
Thanks! This fixes https://github.com/18F/cg-style/issues/202

I also have an approach [here](https://github.com/18F/cg-style/compare/ab/edit-app-limit#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) which ties into the `npm run build` command. If you're developing a new UI component in the library, you're probably already running `npm run watch`, so I think we should tie into that, otherwise you have to run two watches.